commands/rr_capture.jl

```diff
@@ -25,7 +25,7 @@ else
     new_env = copy(ENV)
     mktempdir() do dir
         Pkg.activate(dir)
-        Pkg.add("rr_jll")
+        Pkg.add(Pkg.PackageSpec(name = "rr_jll", version = v"5.4.1"))
```

Comment:
I published 5.5.0+1 earlier today which fixes the issue, so let's use that.

Reply:
Done.